operations/observability/mixins/meta/rules/components/server/alerts.libsonnet

```diff
@@ -49,6 +49,120 @@
               description: 'Server cannot start workspace instances on workspace clusters.',
             },
           },
+          // Rollout alerts
+          {
+            alert: 'JsonRpcApiErrorRates',
+            // Reasoning: the values are taken from past data
+            expr: 'sum (rate(gitpod_server_api_calls_total{statusCode!~"2..|429"}[5m])) / sum(rate(gitpod_server_api_calls_total[5m])) > 0.04',
+            'for': '5m',
+            labels: {
+              // sent to the team internal channel until we fine tuned it
+              severity: 'warning',
+              team: 'webapp'
+            },
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodApiErrorRate.md',
+              summary: 'The error rate of the JSON RPC API is high. Investigation required.',
+              description: 'JSON RPC API error rate high',
+            },
+          },
+          {
+            alert: 'WebsocketConnectionRateHigh',
+            // Reasoning: the values are taken from past data
+            expr: 'sum(rate(server_websocket_connection_count[2m])) > 30',
```

Comment:
Why 2 minutes? That's somewhat non-standard as normally metrics tend to use 1, 3, 5 or 10 minutes

Reply:
Asking out of curiosity: What's the reason behind choosing 1, 3, 5, 10? :thinking: 

fwiw I took 2 after playing around with the numbers and choose the one where the graph looked "nice" (e.g., not too spiky, but responsive enough).